File: atividade-tres-e-quatro/haskell/3/iguais.hs
iguais a b c =
    if a == b && b == c
        then 3
    else if a /= b && b /= c  && a /= c
        then 0
    else 
        then 2

main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    print $ iguais a b c

File: atividade-tres-e-quatro/java/3/Iguais.java
import java.util.ArrayList;

public class Iguais {

       public static void main(String[] args) {

        ArrayList<Int> um = new ArrayList<>();
        ArrayList<Int> dois = new ArrayList<>();
        int c;

        for(Int u : um) {
            for ( Int d : dois) {
                if(um[u] == dois[d]) {
                    c++;
                }
            }
        }
        System.out.println(c);
    }
}
